tigacrafting/views.py

```diff
@@ -1405,15 +1404,6 @@ def auto_annotate_other_species(report, request):
                 current_domain = get_current_domain(request)
                 issue_notification(new_annotation, current_domain)
 
```

Comment:
I don't remember removing this code related to _super_reritja_ !
Sorry if it was me, I will be more cautious from now on..

Reply:
That's weird. Maybe I made these changes after you started working on the pull request? In any case no problem, this is fixed in the pull request review.